README.en.md

```diff
@@ -838,6 +838,47 @@ logging[path] = "/var/logs/error_log.log"
 logging[max_line_length] = 5124
 ```
 
+#### `translate_canonical_tag`
+Configures if WOVN.php should automatically translate existing canonical tag in the HTML. When set to `true`, WOVN.php
+will translate the canonical URL with the current language code. This setting defaults to `true`.
+
+Only language codes and aliases other than the default language will be included in the canonical URL. When the
+default language has an alias, it will be included in the canonical URL.
```

Comment:
This explanation feels unnecessary. The semantics of how lang codes are added isn't really the main part of this setting. Just the fact that we are "translating" canonical urls.

Reply:
also there's a table with "optional parameters" you should update

Reply:
Tentative wording i'm using for wovn.cs:
```
Configures if WOVN.cs should automatically translate existing canonical tag in the HTML. When set to `true`, WOVN.cs
will translate the canonical URL with the current language code according to your `url_pattern` setting. This setting defaults to `true`.

Example:
 `<link rel="canonical" href="/page.html">` may be translated to `<link rel="canonical" href="/en/page.html">` depending on your URL pattern.

```XML
  <wovn>
    <add key="translate_canonical_tag" value="true" />
    ...
  </wovn>
```
```